Update README.md for M1 mac

sshivaji opened the pull request.

sshivaji committed:
Update README.md for M1 mac

lucasccordeiro commented:
Thanks for submitting this PR, @sshivaji.

intrigus-lgtm commented:
@sshivaji can you explain what exactly isn't working on arm Macs?
I just keep hearing that it doesn't work, but I haven't seen a concrete issue that one could fix.
While using amd64 is an easy workaround to any potential problems, I think that it would be best to fix those issues and to provide the best experience to users.
(I don't have any arm Macs)

sshivaji commented:
I think it's fine now. I tried tests on many files, I think it is decent
now, most of the problems were with float_16. I did not know that was a
major thing for CLANG on the M1/ARM Mac.

I appreciate the speed at which we are addressing issues. I will open any
issue I find. We do need to clean up the make build process to make it
easier for new people trying the ARM Mac. Always better to have a native
solution than the complex docker amd64 workaround.

On Wed, 1 Jan 2025 at 05:35, intrigus-lgtm ***@***.***> wrote:

> @sshivaji <https://github.com/sshivaji> can you explain what exactly
> isn't working on arm Macs?
> I just keep hearing that it doesn't work, but I haven't seen a concrete
> issue that one could fix.
> While using amd64 is an easy workaround to any potential problems, I think
> that it would be best to fix those issues and to provide the best
> experience to users.
> (I don't have any arm Macs)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/esbmc/esbmc/pull/2216#issuecomment-2567012302>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAK5IGXWNYNZISSODGLQ7MD2IPVJLAVCNFSM6AAAAABUOPCEVOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDKNRXGAYTEMZQGI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
